conf/machine/include/artik.inc

```diff
@@ -1,13 +1,16 @@
 # Common settings for Samsung Artik boards
+IMAGE_CLASSES += "artik-sdimg"
 
 PREFERRED_PROVIDER_u-boot ?= "u-boot-artik"
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-yocto-artik"
 
 MACHINE_FEATURES += " bluetooth pci rtc ethernet serial usbgadget wifi"
 
 KERNEL_IMAGETYPE = "zImage"
+IMAGE_FSTYPES += "tar.bz2 ext4 artik-sdimg"
 
 MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += "bcm4354-bt bcm4354-nvram-config"
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " kernel-modules"
```

Comment:
Also, can you remove the leading space from " kernel-modules" please? += automatically adds this space